src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -279,9 +283,7 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
                               callbackData.cveId()->str().c_str());
 
                     data->m_elements[callbackData.cveId()->str()] = nlohmann::json::object();
-                    data->m_alerts[callbackData.cveId()->str()]["vulnerability"]["package"]["condition"] =
-                        "Package default status";
-
+                    data->m_matchConditions[callbackData.cveId()->str()] = {"", MatchRuleCondition::DefaultStatus};
                     return true;
                 }
 
```

Comment:
This is not part of this implementation, but the Match by default status does not present any agent information as other match cases. 

```console
2024/03/08 16:31:52 wazuh-modulesd:vulnerability-scanner[55886] packageScanner.hpp:280 at operator()(): DEBUG: Match found for Package: policykit-1 for vulnerability: CVE-2016-2568 due to default status.
```